logic/src/androidMain/kotlin/com/wire/kalium/logic/feature/call/CallManager.kt

```diff
@@ -72,19 +101,39 @@ actual class CallManager(
             incomingCallHandler = { conversationId: String, messageTime: Uint32_t, userId: String, clientId: String, isVideoCall: Boolean,
                                     shouldRing: Boolean, conversationType: Int, arg: Pointer? ->
                 kaliumLogger.i("startHandleAsync -> incomingCallHandler")
+                updateCallStatusById(
+                    conversationId = conversationId,
```

Comment:
It could be based on federation as far as I understood, it is now being handled in `String.toConversationId()` where it verifies if it contains a federation domain or not and it maps down to `ConversationId`